frontend/public/locales/en/public.json

```diff
@@ -1610,5 +1610,8 @@
   "default": "default",
   "OK": "OK",
   "Disabled": "Disabled",
-  "Enabled": "Enabled"
+  "Enabled": "Enabled",
+  "Pipeline status is {{status}}. View logs.": "Pipeline status is {{status}}. View logs.",
+  "Pipeline not started. Start pipeline.": "Pipeline not started. Start pipeline.",
```

Comment:
@rebeccaalpert I added this two messages as aria-label in the topology as "Build decorator". It describes what the icon shows plus what the action "click" does. Is this wording for you or do we have another "guideline" for such a case?

![Screenshot from 2021-06-02 21-37-49](https://user-images.githubusercontent.com/139310/120542015-e368ec80-c3ea-11eb-969e-32abb78711bb.png)

Reply:
Moved it to pipelines -plugin.json, but the question still exists. :)

Reply:
Hi @jerolimov - Let me ping some UXD folks and see what their thoughts are.

Reply:
Hi @jerolimov - Our content team liked these. I also pinged our accessibility expert Jessie Huff and she said she'd take a look.